docs/contribute.md

```diff
@@ -1,34 +1,44 @@
 ## Contribute
-### Picking up issues to complete in existing commands
-1. Fork repo
-2. Create your own branch
-3. `npm install` inside the command directory
-4. Create changes needed in lambda file
-5. Submit pull request
+### Picking up issues to complete in existing commands and features
+1. Leave a comment on the GitHub issue that you want to work on to be assigned to it
+2. Fork repo
+3. Create your own branch
+4. `npm install` inside the command directory
+5. Create changes needed in lambda file
+6. Run unit tests to make sure everything still passes
+7. Create more unit tests if needed
+8. Submit pull request
 
-### Creating new command
+### Creating new Slack slash command
 1. Fork repo
 2. Create your own branch
 3. Create command directory (kebab-case and using the same name as command)
-4. `npm init` and setup new `package.json`
+4. `npm init` and setup new `package.json`, if needed
 5. Create folder called 'cloudformation' to store CloudFormation template
 6. Create Cloudformation template, the following resources are required:
     * Lambda function
     * Lambda IAM role
     * Lambda permission allowing the SNS to call the handler
-    * SNS (SNS name need to match the command in Slack)
+    * SNS Topic (topic name need to match the command in Slack—e.g. the SNS topic for the `health-tm` slash command needs to also be named `health-tm`)
 7. Create Lambda handler
-8. Submit pull request
+8. Create unit tests
+9. Submit pull request
 
 #### CloudFriend and cfn-config
 MapBox has created [Cloudfriend](https://github.com/mapbox/cloudfriend) and [cfn-config](https://github.com/mapbox/cfn-config) to make it easier to create, deploy, and update CloudFormation templates.
 
+### Creating other Slackbot features
+Apart from Slack slash commands, you may be interested in adding other features and integrations to the Slackbot, please submit a GitHub issue for your idea specifying:
+  + What problem your idea is trying to solve
+  + Integrations needed with other tools, APIs etc.
+  + AWS resources needed, if any
+
 ### Testing
-Talk about testing
+// Talk about testing //
 
 
 ### Working with HOT tools
-If you are working with HOT tools such as the Tasking Manager and OSMCha and need a token, please contact _________.
+If you are working with HOT tools such as the Tasking Manager and OSMCha and need a token, please contact //_________//.
```

Comment:
Is this correct? This will be if they want to use tools that HOT has tokens for (TM, OSMCha, HOT Github webhooks etc). Not sure what is best practice is for security.

Reply:
User's TM token can be automatically picked from browser console, but that comes with a set expiry period. Till we set up static token generation feature for the tool, it's good for the contributor to talk to the TM team. For OSMCha, Wille can help generate static tokens. 

So let's say they will have to reach out to either me or Wille for tokens